HISTORY.md

```diff
@@ -1,3 +1,7 @@
+# 3.24.0 / 2020-06-17
```

Comment:
This is an internal-only change so we don't need to update the public changelog. `git checkout master HISTORY.md` should revert this change.